dotCMS/src/main/java/com/dotmarketing/fixtask/FixTasksExecutor.java

```diff
@@ -40,95 +41,84 @@ public static FixTasksExecutor getInstance() {
-    			}
-    		};
-    		try {
-    			runOnce = TaskLocatorUtil.getFixTaskClasses();
-    
-    		} catch (Exception e) {
-    			throw new DotRuntimeException(e.getMessage(), e);
-    		}
-    		Collections.sort(runOnce, comparator);
-    
-    		//PreparedStatement update = null;
-    		int taskId = 0;
-    		try {
-    			for (Class<?> c : runOnce) {
-    				String name = c.getCanonicalName();
-    				name = name.substring(name.lastIndexOf(".") + 1);
-    				String id = name.substring(7, 12);
-
-    				    try{
-    				        taskId = Integer.parseInt(id);
-    				    } catch (NumberFormatException e) {
-    				        taskId++;
-    				    }
-    				    
-    					if (FixTask.class.isAssignableFrom(c)) {
-    						FixTask task;
-    						try {
-    							task = (FixTask) c.newInstance();
-    						} catch (Exception e) {
-    							throw new DotRuntimeException(e.getMessage(), e);
-    						}
-    						HibernateUtil.startTransaction();
-                            Boolean shouldrun=task.shouldRun();
-    						if (shouldrun) {
-    							Logger.info(this, "Running: " + name);
-    							returnValue.addAll(task.executeFix());
-    						}
-    			
-    					String executed="";
-    					if (shouldrun)
-    						executed="was Executed";
-    					else executed="was not Executed";
-    						
-    						Logger.info(this,
-    								"fix assets and inconsistencies task: "
-    										+ name+" "+executed);
-    					}
-    				
-
-    			}
-    		} catch (Exception e) {
-    			Logger
-    					.fatal(
-    							this,
-    							"Unable to execute the fix assets and inconsistencies tasks",
-    							e);
-    			try {
-    				throw new DotDataException(
-    						"Unable to execute fix assets inconsistencies task : ",
-    						e);
-    			} catch (DotDataException e1) {
-    				// TODO Auto-generated catch block
-    			}
-    		}
+			runOnce = TaskLocatorUtil.getFixTaskClasses();
+
+		} catch (Exception e) {
+			throw new DotRuntimeException(e.getMessage(), e);
 		}
-		finally {
-		    try {
-                HibernateUtil.closeSession();
-            } catch (DotHibernateException e) {
-                Logger.warn(this, e.getMessage(), e);
-            }
-		    finally {
-		        DbConnectionFactory.closeConnection();
-		    }
+		Collections.sort(runOnce, comparator);
+
+		//PreparedStatement update = null;
+		int taskId = 0;
+		try {
+			for (Class<?> c : runOnce) {
+				String name = c.getCanonicalName();
+				name = name.substring(name.lastIndexOf(".") + 1);
+				String id = name.substring(7, 12);
+
+					try{
+						taskId = Integer.parseInt(id);
+					} catch (NumberFormatException e) {
+						taskId++;
+					}
+
+					if (FixTask.class.isAssignableFrom(c)) {
+						FixTask task;
+						try {
+							task = (FixTask) c.newInstance();
+						} catch (Exception e) {
+							throw new DotRuntimeException(e.getMessage(), e);
+						}
+						HibernateUtil.startTransaction();
```

Comment:
Not sure what are u doing here?

Reply:
Removed the finally statement. Seems identation changed, 
Look at the file ignoring whitespaces:
https://github.com/dotCMS/core/pull/13990/files/1953623013e7993a8b59d3a067eaa33174a458da?w=1#diff-828efa25304c59a68cf8635e6e9f6f2b